### DISCUSSION TITLE
Floating input label

### DISCUSSION THREAD

**Discussion by nathanmerry** [saadeghi/daisyui#2239] [Ideas / feature requests]
I think it would be really nice for a floating input label feature. Floating input labels look so much cleaner in my opinion than the standard input label we have currently

Many other libraries, such as vuetify or tailwind-elements support this feature so its a shame we don't see it here!

**Comment by nathanmerry** [comment]
And i suppose this would be a really nice feature for all for elements also!

**Comment by saadeghi** [comment]
duplicate of https://github.com/saadeghi/daisyui/issues/298
